workers.go

```diff
@@ -335,10 +334,10 @@ func (chp *GubernatorPool) handleGetRateLimit(handlerRequest *request, cache Cac
 	}
 }
 
-// Atomically load cache from persistent storage.
+// Load atomically loads cache from persistent storage.
 // Read from persistent storage.  Load into each appropriate worker's cache.
 // Workers are locked during this load operation to prevent race conditions.
```

Comment:
```suggestion
// Load atomically loads cache from persistent storage into each appropriate worker's cache. 
// Workers are locking during this step to avoid race conditions
```

Feels like this comment block is a bit repetitive. Maybe replacing the block with something like this could read a bit more succinctly?